test/core/utils/preload-cssom.js

```diff
@@ -0,0 +1,120 @@
+describe('axe.utils.preloadCssom', function () {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var pageTpl = '<!DOCTYPE html><html><head><style>html {width: 100vh;}</style></head><body></body></html>';
+
+	afterEach(function () {
+		fixture.innerHTML = '';
+		axe._tree = undefined;
+	});
+
+	it('should be a function', function () {
+		assert.isFunction(axe.utils.preloadCssom);
+	});
+
+	it('should return a queue', function () {
+		fixture.innerHTML = pageTpl;
+		var tree = axe._tree = axe.utils.getFlattenedTree(fixture);
+		var args = {
+			asset: ['cssom'],
```

Comment:
Why do we need to tell a CSSOM loader that it's loading CSSOM? That seems unnecessary.